README.md

```diff
@@ -283,6 +283,19 @@ juju config kserve-controller http-proxy=http://10.0.13.50:3128/ https-proxy=htt
 juju config knative-serving http-proxy=http://10.0.13.50:3128/ https-proxy=http://10.0.13.50:3128/ no-proxy=10.1.0.0/16,10.152.183.0/24,127.0.0.1,localhost,10.0.2.0/24,ip-10-0-2-157,.svc,.local
 ```
 
+A helpful set of commands to achieve this in a Microk8s deployment is:
```

Comment:
really nice improvement to the UX, I suggest we make this the default instruction to run the UATs behind proxy not just as a helpful hint, and same for the [PodDefault section in running using Notebook](https://github.com/canonical/charmed-kubeflow-uats/blob/main/README.md#prerequisites-2). wdyt?

Reply:
As we discussed, I took the comment as a start and refactored the whole section for a more streamlined and easy to follow instructions. I also renamed the PR. LMKWYT.